spec/javascript/components/molecules/InlineModal.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import InlineModal from 'components/molecules/InlineModal'
+
+describe('InlineModal', () => {
+  test('should use the inline-modal class for background if background is not specified', () => {
+    const wrapper = mount(<InlineModal />)
```

Comment:
`shallow` works just as well for these right? I prefer we always default to shallow unless necessary

Reply:
oh yep! I thought it wouldn't show the classnames but it does. Switched

Reply:
@akegan yeah `shallow` shows all props, and you can think of `classNames`, `style`, etc as props, since that's literally how they're implemented. They're not some special keywords or anything

Reply:
thanks!